### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by MShirishaReddy1** [community/community#46105] [Copilot Conversations]
I am using free Trail Period but I am unable to use copilot on vs code. Could u Please Help Me out with this?

![image](https://user-images.githubusercontent.com/123538128/216254843-3fe2af44-0825-4c45-aa30-0c32ffd425c3.png)

**Comment by DanteHaxor** [comment]
this is an bug which was caused by a bad deploy which affected a small percentage of Copilot users. The change has been rolled back and you should be able to use Copilot again

**Reply by MShirishaReddy1** [reply]
But the issue still persists.

**Reply by singampk** [reply]
I still face the same issue, tried many times. I am firsttime user, subscribed last night.
- IntelliJ
- MacOS

**Reply by sedentaryOne** [reply]
Still seems like a bug that's affecting a percentage of users an you have no idea why.  That's such a PR answer to a problem.

**Comment by davecheney** [comment]
@MShirishaReddy1 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by UniBreakfast** [reply]
didn't help

**Comment by UniBreakfast** [comment]
Same issue here. Year plan. Worked fine for many months. Not today though.(((

**Comment by innokentiy2517** [comment]
Same issue. Says "Contact support"

**Comment by Kseen715** [comment]
Same issue. Seems like it can't connect to the server.

**Comment by potdmk** [comment]
Same issue as well. 
I tried to sign-out, reinstall extension, change device - still get this error.

**Comment by almazmusic** [comment]
Same thing. Log out didn't help. Created a support ticket.

**Comment by yaBobJonez** [comment]
Same issue for me, but on JetBrains CLion and Rider.
Just activated Student Pack and installed the Copilot plugin, but I can't log in, it shows "Contact support".

**Reply by yaBobJonez** [reply]
Update: looks like GitHub banned access to Copilot for Ukrainian IPs as many users on the Internet suspected, although it seems like currently, Copilot started working for me again!

**Comment by prado1104** [comment]
Any news? Same problem.

**Comment by powernic** [comment]
Same issue

**Reply by peelmicro** [reply]
It was working for me for many days, and today stopped working with the same error
